prow/e2e-cluster_wide-auth.sh

```diff
@@ -31,4 +31,4 @@ set -x
 ROOT=$(cd $(dirname $0)/..; pwd)
 
 echo 'Running cluster-wide e2e rbac, auth Tests'
-${ROOT}/prow/e2e-suite.sh --test_vm --auth_enable --cluster_wide --use_galley_config_validator "$@"
+${ROOT}/prow/e2e-suite.sh --test_vm --auth_enable --cluster_wide --use-automatic-injection --use_galley_config_validator "$@"
```

Comment:
Shouldn't we have separate suites, one with automatic one with manual ? Or do we already have ?